iceoryx_hoofs/include/iceoryx_hoofs/cxx/unique_ptr.hpp

```diff
@@ -48,15 +48,15 @@ class unique_ptr
     unique_ptr(const unique_ptr& other) = delete;
     unique_ptr& operator=(const unique_ptr&) = delete;
     unique_ptr(unique_ptr&& rhs) noexcept;
-    unique_ptr& operator=(unique_ptr&& rhs) noexcept;
+    unique_ptr& operator=(unique_ptr&& rhs) & noexcept;
```

Comment:
As far as I know the decision is to disable the rule so this is not needed anymore.